src/components/dialogs/reccuring-closure-config-dialog/recurring-modes/interval-recurring-mode/IntervalConfigForm.tsx

```
@@ -110,17 +110,17 @@ export function IntervalConfigForm(props: RecurringModeFormProps) {
 
       <wz-caption>
         {t('edit.closure.recurrence.interval.recurrence_explanation', {
-          closureDuration:
+          duration:
             isFinite(closureDuration) ?
               formatMinutes(closureDuration)
             : `<${t('edit.closure.recurrence.interval.closure_duration_label')}>`,
-          intervalBetweenClosures:
+          interval:
             isFinite(intervalBetweenClosures) ?
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)